boxsdk/object/web_link.py

```diff
@@ -0,0 +1,11 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
```

Comment:
Also add `absolute_import`

Add a blank line between the `__future__` and other imports.